### PR TITLE
fix proposal prefix

### PR DESCRIPTION
**Pull request by wangchao0222**
## Description

Solve the problem that some user tokens are not unlocked after the proposal ends

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Brief of your solution

Please describe your solution to solve the issue or feature request.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

### CODE REVIEW

File: kernel/contract/proposal/utils/utils.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func MakeProposalLockPrefix(proposalID string) string {
 
 // MakeTimerBlockHeightPrefix generate timer and blockHeight prefix
 func MakeProposalLockPrefixSeparator(proposalID string) string {
-	return proposalLockKey + separator + proposalID + separator + separator
+	return proposalLockKey + separator + proposalID + separator + "~"
```
Comment (godeamon) [inline kernel/contract/proposal/utils/utils.go:96]: 会和老数据不兼容吗？

Comment (zheng-bin) [inline kernel/contract/proposal/utils/utils.go]: "~" 做成const，起个名字？

**Review by godeamon — APPROVED** [review]

File: kernel/contract/proposal/utils/utils.go (hunk 1 above)
Comment (wangchao0222) [reply kernel/contract/proposal/utils/utils.go]: 兼容的，这里仅仅是计算迭代的结束条件

**Review by zheng-bin — APPROVED** [review]
LGTM

**Pull request merged**